tardis/apps/publication_workflow/static/js/lib/angular-messages-1.6.10/angular-messages.js

```diff
@@ -0,0 +1,747 @@
+/**
+ * @license AngularJS v1.6.10
+ * (c) 2010-2018 Google, Inc. http://angularjs.org
+ * License: MIT
+ */
+(function(window, angular) {'use strict';
```

Comment:
Codacy has a fix for the issue: ['use strict' is unnecessary inside of modules.](https://app.codacy.com/gh/mytardis/mytardis/pullRequest?prid=7933947)
```suggestion
(function(window, angular) {
```